src/my-view1.html

```diff
@@ -193,7 +193,9 @@
     <div class="content">
       <div class="description">
       <h2 class="intro-description center color-secondary">
-        Create an email to request a free bike corral from the City of Austin
+        <strong>Make your voice heard!</strong> 
```

Comment:
This tagline sounds a bit too much like activists. Although I like the idea, I believe we should stick to a simple tagline which conveys the message that the bike rack request process is now easy.

Reply:
Okay. I'll update it now